staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -167,7 +167,7 @@ func (rt *userAgentRoundTripper) WrappedRoundTripper() http.RoundTripper { retur
 
 type basicAuthRoundTripper struct {
 	username string
-	password string
+	password string `datapolicy:"password"`
```

Comment:
Far below, `requestInfo.RequestHeader` should be marked?  It was responsible for `CVE-2020-8565`.